articles/integrations/azure-api-management/configure-auth0.md

```diff
@@ -0,0 +1,73 @@
+
+To begin, you'll need to log into the Auth0 [Management Dashboard](${manage_url}). Navigate to **APIs**, and click on **Create API**.
+
+![](/media/articles/integrations/azure-api-mgmt/auth0/apis.png)
+
+You'll be asked to provide values for the following parameters before Auth0 creates your new API:
+
+| Parameter | Description |
+| --------- | ----------- |
+| Name | A descriptive name for your API |
+| Identifier | A logical identifier for your API |
+| Signing Algorithm | The method used to sign the tokens issued by Auth0 |
+
+![](/media/articles/integrations/azure-api-mgmt/auth0/new-api-config.png)
+
+When complete, click **Create**.
+
+![](/media/articles/integrations/azure-api-mgmt/auth0/api-config.png)
+
+When your API is ready, you'll be shown the **Quick Start** page for the API. Switch over to the **Non Interactive Clients** tab. You'll see that Auth0 has also created and enabled a non interactive [client](/clients) for use with your API.
+
+![](/media/articles/integrations/azure-api-mgmt/auth0/api-nic.png)
+
+### Step 2: Create a Connection
+
+After you've created your API and your Client, you'll need to create a [Connection](https://auth0.com/docs/clients/connections), which are sources of users. For the purposes of this example, we'll create a [Database Connection](/connections/database).
+
+To begin, you'll need to log into the Auth0 [Management Dashboard](${manage_url}). Navigate to **Connections** > **Database Connections**, and click on **Create DB Connection**.
+
+![](/media/articles/integrations/azure-api-mgmt/auth0/db-connections.png)
+
+The only thing you'll need to provide at this time is a descriptive **Name** for your connection.
+
+![](/media/articles/integrations/azure-api-mgmt/auth0/new-db-connection-config.png)
+
+Click **Create** to proceed.
+
+#### Enable the Connection for Your Client
+
+Once Auth0 has created your Connection, you'll be redirected to your Connection's **Settings** page. Switch over to the **Clients** tab. You'll need to enable the Connection for use with the Non Interactive Client that you're using with your API.
+
+![](/media/articles/integrations/azure-api-mgmt/auth0/connection-client.png)
+
+### Step 3: Create a User
+
+Finally, we will create a user stored in your Connection. You'll use this user's credentials at a later point to test your integration.
+
+To begin, you'll need to log into the Auth0 [Management Dashboard](${manage_url}). Navigate to **Users**, and click on **Create User**.
+
+![](/media/articles/integrations/azure-api-mgmt/auth0/users.png)
+
+Provide an **email** and **password** for your new user. Be sure to indicate that this user should use **BasicCalculator** in the **Connection** field.
```

Comment:
I'd change the last sentence to sth like: Set the **Connection** field to the connection you created earlier (`BasicCalculator` if you are following along our example).

(Also, avoid `should`)